### PR TITLE
Replace logrus zerolog

### PR DESCRIPTION
**Pull request by Richard87**

### CODE REVIEW

File: pkg/utils/logger/logger.go [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+func InitializeLogger(ctx context.Context, logLevel zerolog.Level, prettyPrint bool) context.Context {
+	zerolog.SetGlobalLevel(logLevel)
+	zerolog.DurationFieldUnit = time.Millisecond
+	if prettyPrint {
+		log.Logger = log.Output(zerolog.ConsoleWriter{Out: os.Stderr, TimeFormat: time.TimeOnly})
+	}
+	ctx = log.Logger.WithContext(ctx)
+
+	log.Debug().Msgf("log-level '%v'", logLevel.String())
+	return ctx
```
Comment (nilsgstrabo) [inline pkg/utils/logger/logger.go:21]: ctx is not used by caller. Should we remove it?

File: main.go [hunk 2]
```diff
@@ -1,42 +1,46 @@
...
 
 func main() {
 	environment := env.NewEnvironment()
-	setLogLevel(environment)
+	level, err := environment.GetLogLevel()
+	if err != nil {
+		log.Fatal().Err(err).Msg("Failed to initialize log level")
+	}
+	logger.InitializeLogger(context.Background(), level, true)
+	if err != nil {
```
Comment (nilsgstrabo) [inline main.go:23]: This `err` is not returned by InitializeLogger(), but comes from GetLogLevel()

**Review by nilsgstrabo — CHANGES_REQUESTED** [review]
Added a few comments

File: pkg/utils/logger/logger.go (hunk 1 above)
Comment (Richard87) [reply pkg/utils/logger/logger.go:21]: agreed, and fixed

File: main.go (hunk 2 above)
Comment (Richard87) [reply main.go:23]: :O  Fixed

**Pull request merged**